README.md

```diff
@@ -105,8 +105,25 @@ Yamashita | Yah-MAH-sh-Ta  | some vowels (i, u) get removed (called devoicing),
 Kyoko     | Kyoh-koh       | (not Kee-oh-koh)      
 Ryousuke  | Ryoh-skeh      | the -suke suffix is usually skeh
 
+## Portuguese (Brazilian)
+
+The challenge with Portuguese is that there are some phonems (such as \[ɐ̃w̃\], \[s̻\] or \[ʎ\]) and [diacritics](https://en.wikipedia.org/wiki/Diacritic) (such as an accent or cedilla) not common or not at all present in the English language. 
+
+First, a few general rules:
+1) **ão** is a nasal diphthong and resembles "pawn" + "o"
+2) **ç** is like a soft S sound (think of how you'd prounounce the French word [facade](https://en.wiktionary.org/wiki/facade))
+3) **lh** is similar to the Spanish "ll" as in "pollo" (chicken) or "pol" - "io"
+4) **nh** is similar to the Spanish "ñ", as in "piña" (pineapple) or "pin" - "a"
+
+Here are some common names and surnames in alphabetical order and how to pronounce them:
+Name (in **bold** is the stressed syllable) | Pronunciation Tricks
+--------------------------------------------------------|---------------------
+Gui**lher**me | this is tricky, Gui as in "gui-tar" - lher, as in "lee-er" - "me"
+Jo**ão** | Jo, this is a soft J, so more like "sho-w" than like "Joh-n" - ão, which is a nasal sound, almost as in "pawn" + "o"
+Men**don**ça (surname) | "Men" - "don" - sa, as in "Sa-rah"
+Ronal**din**ho | "Ro" - "nal" - "din" - "o"
 
 ---
-by Steve Carroll
+created by Steve Carroll (ja, zh) and with contributions from Carlos Mendonça (pt-br)
```

Comment:
yeah... i wasn't expecting to get contributions so quickly.   I'll create an issue to make a contributors section.